### DISCUSSION TITLE
How to "break" text if it is too long ?

### DISCUSSION THREAD

**Discussion by itishermann** [mantinedev/mantine#1997] [Q&A] [answered]
Given this component
```jsx
<UnstyledButton>
  <Group>
    <Avatar size={40} color="blue">BH</Avatar>
    <div>
      <Text>Bob Handsome</Text>
      <Text size="xs" color="dimmed">bob@handsome.inc</Text>
    </div>
  </Group>
</UnstyledButton>
 ```

How to "break" the email inside the div if the email is too long ?
instead of having a super long email, we get 'super.long.email@gm..."
Any hint? Thanks in advance.

**Comment by rtivital** [comment]
Add word-break styles – https://developer.mozilla.org/en-US/docs/Web/CSS/word-break

**Comment by sqveeze** [comment, marked as answer]
Hey, on the text component you can use:

https://mantine.dev/core/text/#line-clamp
